### ISSUE TITLE
WCAG 2.1 Tooltip escaping

### ISSUE THREAD

**Issue by qoalu** [reach/reach-ui#202]
https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus

Level AA Success criterion:

> **Dismissable**
> A mechanism is available to dismiss the additional content without moving pointer hover or keyboard focus, unless the additional content communicates an input error or does not obscure or replace other content;

Does this mean that the tooltip should be dismissable on ESC key press?

**Comment by chancestrickland** [comment]
@qoalu You are correct here. We do need to add the ability to dismiss tooltips that appear on hover (they should be currently dismissable on focus, so it's just a matter of expanding that).

**Comment by jescowuester** [comment]
Would love to take this one on

**Comment by jescowuester** [comment]
woud love to take this one on

**Comment by chancestrickland** [comment]
@jescowuester Have at it!
